src/js/LineChart.jsx

```diff
@@ -0,0 +1,294 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+  setDuration,
+  setCoin,
+  setHoverLoc,
+  setActivePoint
+} from './actionCreators';
+import './LineChart.css';
+
+class LineChart extends Component {
+  constructor(props) {
+    super(props);
+  }
+  // GET X & Y || MAX & MIN
+  getX() {
+    const { data, duration, coin } = this.props;
+    const lookAt = data[coin];
+    return {
+      min: lookAt[duration][0].x,
+      max: data[coin][duration][data[coin][duration].length - 1].x
```

Comment:
What happens if any of these variables are `undefined`?

Reply:
:1st_place_medal: